### DISCUSSION TITLE
How to get the number of windows in a tab?

### DISCUSSION THREAD

**Discussion by luiz00martins** [nanozuki/tabby.nvim#107] [Q&A] [answered]
I tried the following:
```lua
line.tabs().foreach(function(tab)
    local wins = tab.wins()
    local win_count = #wins
    
    ...
```

but that always returns me 0.

**Comment by nanozuki** [comment, marked as answer]
Sorry for my lateness, I was so busy in the last two months.

The `tab.wins` is not an array, this object has `wins` and `foreach` two attributes. So, you can get the windows' count, you can use:

```lua
local win_count = #wins.wins
```
Or other low level way:

```
local api = require('tabby.module.api')
local wins = api.get_tab_wins(tabid)
local win_count = #wins
```
